### ISSUE TITLE
[Bug] Store as attribute' does not work for sizers in C++ [sf#192]

### ISSUE THREAD

**Issue by DietmarSchwertberger** [DietmarSchwertberger/wxGlade#267]
**Reported by eanon on 2016-07-12 11:14 UTC**
Hello. When I enable "store as attribute" for a sizer, the type declaration remains in the implementation file (.cpp). So, this variable becomes local and shadows the real declaration as member in the header file (.h).

Example:
~~~
// In do_layout() implementation (.cpp):
wxBoxSizer* sizerForIdentity = new wxBoxSizer(wxVERTICAL);

// In protected members of class declaration (.h):
wxBoxSizer* sizerForIdentity;
~~~

**Comment by DietmarSchwertberger** [comment]
**Commented by cgrohmann on 2016-11-05 21:17 UTC**
Hi,

the bug is fixed in the development branch with 2182:f1f66815ed4c "Fix 'Store as attribute' does not work for sizers in C++ (sf bug #192)"

The change will be also part of the next official release.

Thank you for reporting this bug.
Carsten

**Comment by DietmarSchwertberger** [comment]
**Updated by cgrohmann on 2016-11-05 21:18 UTC**
- **summary**: Still declared as local variable in spite of 'store as attribute' option --> Store as attribute' does not work for sizers in C++
- **status**: open --> closed
- **assigned_to**: Carsten Grohmann
